README.md

```diff
@@ -142,3 +142,30 @@ an input and returns a str. By default, `RichHelpFormatter` sets the function to
 ```python
 RichHelpFormatter.group_name_formatter = str.title
 ```
+
+## Contributing
+Start by [forking the repo](https://github.com/hamdanal/rich-argparse/fork) and cloning your fork.
+
+For those using [VS Code](https://code.visualstudio.com/) there's [preconfigured settings](.vscode/) in the repo.
+
+### Environment Setup
+You can use `tox` to setup a virtual environment for `rich_argparse` and install dependencies.
```

Comment:
Here I think a link to tox is useful since some people might not be familiar with it

Reply:
👍